### ISSUE TITLE
Tooltips are behaving differently compared to plain Vaadin 7

### ISSUE THREAD

**Issue by mlopezFC** [vaadin/multiplatform-runtime#111]
In Vaadin 7, tooltips had the following behavior:
* When the mouse cursor was hovering the element that produced the tooltip, then the tooltip appeared
* If the mouse pointer was moved to hover on top of the tooltip, it remained opened and you could even click on it and the tooltip won't be closed
* When moving the mouse out of the element or the tooltip, then the tooltip disappeared.
This behavior can be seen in the following animated git that was created using a sample Vaadin 7 project that I'm attaching (plain-vaadin-7.zip):

![Vaadin7](https://user-images.githubusercontent.com/28312758/167484463-cf403ee6-0c94-47ab-a9d5-fe148aee85f3.gif)
[plain-vaadin-7.zip](https://github.com/vaadin/multiplatform-runtime/files/8654840/plain-vaadin-7.zip)

After migrating the project to Vaadin 7 + MPR 2.2.2 then the behavior changed a little bit, and was reported by [this closed issue](https://github.com/vaadin/multiplatform-runtime/issues/89).
Anyway the behavior regarding the mousepointer and clicking on the tooltip remained the same.
This animated gif shows how it behaved with that version (MPR 2.2.2):
![Vaadin7MPR222](https://user-images.githubusercontent.com/28312758/167484906-58d0546f-6897-4943-ab16-5581b429b9bc.gif)
[mpr-v7-2.2.2.zip](https://github.com/vaadin/multiplatform-runtime/files/8654845/mpr-v7-2.2.2.zip)

After that fix was released, the behavior changed. Now it's not possible to even click on the tooltip anymore, it will close automatically when the mouse is hovering over the tooltip as seen in the following gif (MPR 2.2.6):

![Vaadin7MPR226](https://user-images.githubusercontent.com/28312758/167485139-955f70d1-da1f-4e7c-a399-ef1689c477a8.gif)
[mpr-v7-2.2.6.zip](https://github.com/vaadin/multiplatform-runtime/files/8654852/mpr-v7-2.2.6.zip)

So basically now with the latest MPR version the tooltips have a different behavior compared to plain Vaadin 7.

**Comment by TatuLund** [comment]
Noting here, that if you have a very complex description with html content, it becomes more like a popup that opens on hover. In that case it becomes a necessity to be able to scroll the content of the tooltip. Example:

```
String description = "<b>This searches:</b>\n" +
        "<ul>\n" +
        "\t<li>Product: Product Name</li>\n" +
        "\t<li>Product: Product Number</li>\n" +
        "\t<li>Product: GTIN Number</li>\n" +
        "\t<li>Product: UPC Number</li>\n" +
        "\t<li>Lumpsum: Lumpsum Description</li>\n" +
        "\t<li>Lumpsum: Lumpsum CD</li>\n" +
        "\t<li>Contract Lumpsum: Contract Lumpsum Description</li>\n" +
        "</ul>";

field.setDescription(description);
```

**Comment by davemc-telus** [comment]
This is one of our highest priority bugs. Our customers are frequently asking when this can be fixed. Thanks for looking into it.

**Comment by Tostino** [comment]
Where are we with this? I haven't heard any progress and we're about to pass another release for our customers without having any resolution here...

**Comment by mshabarov** [comment]
@Tostino we'll start working on this ticket as soon as possible. Since we have quite many critical issues in our queue, we plan to start this work on the next iteration (next week).

**Comment by roastedcpu** [comment]
Any updates on this one ?

**Comment by mcollovati** [comment]
A new version with the fix for this issue will be released soon

**Comment by mcollovati** [comment]
Fix is available on 6.0.2, 5.0.9, 2.2.10 and will be part of next Vaadin platform releases
